theseus/geometry/functional/tests/common.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+
+
+def check_lie_group_function(module, method: str, atol: float, *args):
```

Comment:
Here it is also clearer to directly pass the functions you are testing. Also, I don't quite understand the difference between `func_call` and `func_apply`. I'm guessing this names are remnants from the initial implementation?

Reply:
Rename some of the variables, and the differences are explained in https://github.com/facebookresearch/theseus/blob/86ea13facd1acfb46b522f0aa6ddfcb4de71662d/theseus/geometry/functional/lie_group.py#L12